index.html

```diff
@@ -192,6 +192,64 @@ <h3><dfn>Source</dfn> interface</h3>
       </p>
     </section>
 
+    <section data-dfn-for="Range">
+      <h3><dfn>Range</dfn> interface</h3>
+      <pre class="idl">
+      interface Range {
+        attribute gt? Term;
+        attribute gte? Term;
+        attribute lt? Term;
+        attribute lte? Term;
+      };
+      </pre>
+      <p>
+        A Range is an object that defines a gamut of values by declaring the
+        gamut's own inclusive or exclusive boundaries.
+      </p>
+    </section>
+
+    <section data-dfn-for="RangeSource">
+      <h3><dfn>RangeSource</dfn> interface</h3>
+      <pre class="idl">
+      [Constructor,
+       Constructor(ConstructorOptions options)]
+      interface RangeSource {
+        Stream&lt;Quad&gt; match(
+          optional Term|Range[]|Range? subject,
+          optional Term|Range[]|Range? predicate,
+          optional Term|Range[]|Range? object,
+          optional Term|Range[]|Range? graph
+        );
```

Comment:
When one would use Range at any other position in the quad than `object` ? I think it should only apply to Literals which only can appear on `object` position.

Reply:
I have never had to do it myself but I've seen prefix-based queries targeting `subject` and `graph` named nodes a few times.